### ISSUE TITLE
Threads display problem

### ISSUE THREAD

**Issue by a-stark** [dokufreaks/plugin-discussion#223]
Hi there,
I try to display separate discussion in different name spaces and the threads command seems not to work for me. Here is my configuration:

I have in the namespace `log` different kind of user pages (i.e. different kind of users), e.g.
 ```
log:user1:<user-defined-logs>
log:user2:<user-defined-logs>
...
```
where `<user-defined-logs>` are pages, which are created by the [yearbox plugin](https://www.dokuwiki.org/plugin:yearbox). Basically, `log:user1:<user-defined-logs>` contains some further pages, where each page contains a `~~DISCUSSION~~` keysword. 

Everything works fine and each user can comment on every page. Now I want to display for each user a summary of threads for their namespace. For instance, for user `user1` on the page
`log:user1:calendar` all threads should be mentioned in the namespace `log:user1:*`. Therefore I make an entry on the page `log:user1:calendar` with

`{{threads>log:user1:*}}`

If I am logged in as admin, this works fine, I can see there exactly the threads corresponding to the namespace `log:user1:*` like in this picture:

![grafik](https://cloud.githubusercontent.com/assets/18192374/24083640/13385ade-0cdb-11e7-8aa4-a9eca1d5d516.png)

But if I am logged in as `user1`, I can only see that:

![grafik](https://cloud.githubusercontent.com/assets/18192374/24083632/e891aa24-0cda-11e7-8e7d-e0891d9f523f.png)

I have selected the option `threads_formposition:off`, so I'm actually not expecting to see the new discussion page. That is the first strange behavior.

The second is, that the user1 is not able to see the threads in this namespace, just the admin. 
If I add the user and the admin to a group `mods` (moderators) and add this group to the options `moderatorgroups` I see no change.
I have also made sure, that the namespace `log:user1:*` has all right for `user1` to create and delete (sub)pages.

I have the feeling that the authentication for the respective user does not work properly. Or do I use the plugin in a wrong way? I would be happy about any suggestions.

**Comment by a-stark** [comment]
Ah I forgot to mention that if I use `{{threads>*}}` or `{{threads>:}}` I got an display of threads if I am logged in as `user1`, but for all user namespaces, i.e. for `log:user1:*`, `log:user2:`, ... which I want to prevent. I just want to display the discussion for the specific user. Moreover `{{threads>.}}` does not work and displays the above shown 'New discussion page'.

**Comment by Klap-in** [comment]
I think this is the same issue as #330 (with PR #331 of @Vincent31Fr), which is implemented with commit ab18f68d213366961a41faba15955c6e35dab2f4 (part of PR #333).

I understood permissions are not correctly applied for non-admins in the search action.

So I hope it is solved by #333?

**Comment by Klap-in** [comment]
If not yet fixed, please reopen.
